### ISSUE TITLE
chore(ssz): Lift bellatrix ssz file into git LFS

### ISSUE THREAD

**Issue by chibera** [berachain/beacon-kit#1152]
We currently ship a 50mb bellatrix ssz serialized block with the beacon-kit.   
This file is used for `spec_test.go` in ssz v2
As we add #810 it will add even more serialized binary blobs. 

After #810 lift all binary files into git LFS or exclude them from the final binary some other way.

**Comment by shotes** [comment]
No longer relevant. File does not exist.
